master/types.html

```diff
@@ -144,7 +144,11 @@ <h2 id="syntax">Attribute syntax</h2>
 unitless length and angles to be used in presentation attribute while
 disallowing them in corresponding property values.</p>
 
-<p class="note">Note that all <a>presentation attributes</a>, since they are
+<p>The <a>'d'</a> presentation attribute is an exception, it is parsed
```

Comment:
For CSS Transforms we are a little bit more verbose and explain why there are differences as well. See https://drafts.csswg.org/css-transforms/#svg-syntax I think we should be a little bit more verbose here too.

Did we agree on freezing the `d` presentation attribute to the definition of SVG 1.1? No support of the additional CSS property values?

IMO we should allow all values the property has + the string value itself for backwards compatibility reasons. But I am not sure if:
1. We have discussed this in the WG already and resolved to not extend the grammar
2. Chrome would support the additional CSS syntax on the attribute as well.

Reply:
Guess I commented here already. Still applies IMO.

Reply:
I think it would confuse parsing too much to to allow the `path()` notation inside the attribute.

That said, I don't see any reason to freeze the attribute at SVG 1.1 when it comes to path data commands. Instead, treat the attribute value as a string injected into a `path()` wrapper. If we ever do add other commands, they would be valid here, too.